content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md

```diff
@@ -40,30 +40,40 @@ It's good practice to structure your files well, so we'll expect you to have a s
 └──.gitignore 
 ```
 
+## Dealing with missing values in a dataset
+
+Before we dive into the tasks of this project, let's talk a bit about missing values. For a variety of reasons, the data we work with could have missing or invalid data. Depending on the kind of data we are working with and the extent of the data missing, we typically deal with missing data either by dropping it (for example in the case of entirely/nearly entirely missing columns or rows) or by replacing them with a suitable values.
+
+Later in this program you'll be learning how to train machine learning models using various kinds of data. When doing so, missing values cannot simply be ignored. Further, when it comes to datasets with many columns (also known as features), it is bad practice to drop an entire row for a few missing values. 
+
+Here is a resource for dealing with missing values : https://www.analyticsvidhya.com/blog/2021/10/a-complete-guide-to-dealing-with-missing-values-in-python/
+
 
 ## Questions
```

Comment:
Are these questions? Or instructions?

Reply:
There are some instructions and some questions, but happy to change it to instructions